src/map/ai/ai_mob_dummy.cpp

```diff
@@ -408,18 +408,18 @@ void CAIMobDummy::ActionDropItems()
 				blueutils::TryLearningSpells(PChar, m_PMob);
 				m_PMob->m_UsedSkillIds.clear();
 
-				if(m_PMob->m_giveExp)
-				{
-	                charutils::DistributeExperiencePoints(PChar, m_PMob);
-				}
+                if(m_PMob->m_giveExp && m_PMob->getMobMod(MOBMOD_EXP_BONUS) >= 0)

```

Comment:
just double checked...I get exp just fine off the water elemental next to my test subject (a flamingo I set to not give exp via the mod)